Files/Views/Settings.xaml.cs

```diff
@@ -29,6 +30,8 @@ protected override void OnNavigatedTo(NavigationEventArgs eventArgs)
             var currentView = SystemNavigationManager.GetForCurrentView();
             currentView.BackRequested += OnBackRequested;
 
+            Window.Current.CoreWindow.Dispatcher.AcceleratorKeyActivated += CoreDispatcher_AcceleratorKeyActivated;
```

Comment:
Don't forget to unsubscribe from this event when the user leaves the page. Another option would be to register this from the xaml.